patches/minecraft/net/minecraft/block/NetherPortalBlock.java.patch

```diff
@@ -45,3 +45,64 @@
                 this.field_150862_g = 0;
                 break;
              }
+
+@@ -217,19 +217,25 @@ public class NetherPortalBlock extends Block {
+    }
+
+    public static class Size {
+-      private final IWorld world;
+-      private final Direction.Axis axis;
+-      private final Direction rightDir;
+-      private final Direction leftDir;
+-      private int portalBlockCount;
++      protected final IWorld world;
++      protected final Direction.Axis axis;
++      protected final Direction rightDir;
++      protected final Direction leftDir;
++      protected int portalBlockCount;
+       @Nullable
+-      private BlockPos bottomLeft;
+-      private int height;
+-      private int width;
++      protected BlockPos bottomLeft;
++      protected int height;
++      protected int width;
++      protected Block portalBlock;
+
+       public Size(IWorld worldIn, BlockPos pos, Direction.Axis axisIn) {
++         this(worldIn, pos, axisIn, Blocks.NETHER_PORTAL);
++      }
++
++      public Size(IWorld worldIn, BlockPos pos, Direction.Axis axisIn, Block portalBlock) {
+          this.world = worldIn;
+          this.axis = axisIn;
++         this.portalBlock = portalBlock;
```

Comment:
Move this assignment up (first thing in the constructor) to get a consecutive patch.